.github/workflows/labeler.yml

```diff
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
```

Comment:
Let's enumerate the events we want to use as triggers to avoid unnecessary runs. Probably: opened, reopened, and synchronized (i.e., when someone pushes to the PR)